### ISSUE TITLE
octool_rpi.sh broken

### ISSUE THREAD

**Issue by Dagiopia** [opencog/ocpkg#117]
variables in src files set by cmake files disturb running when running on a pi. 
this was a long running problem but since the focus was ghost there was no need for loading modules from the cogserver but now since ghost needs modules, this needs to be fixed.

**Comment by Dagiopia** [comment]
There's a deb [here](http://community.goertzel.org/opencog/opencog-dev_1.0-1_armhf.deb) that is fine.
Updated versions will always be tagged with cogutil,atomspace and opencog commits which one can see with dpkg --info and also after installation, the commits can be found from pkg-config vars cogutil,atomspace and opencog.

**Comment by AmeBel** [comment]
@Dagiopia  did you use https://github.com/jordansissel/fpm or did you author the package configuration files?

**Comment by Dagiopia** [comment]
yeah I wrote them. We only need debian packages so, dpkg-deb works fine. Didn't use fpm.
